[2025-10-16T09:55:47.567+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T09:55:47.582+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T09:54:00+00:00 [queued]>
[2025-10-16T09:55:47.587+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T09:54:00+00:00 [queued]>
[2025-10-16T09:55:47.638+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T09:55:47.720+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dag2_bash_1> on 2025-10-16 09:54:00+00:00
[2025-10-16T09:55:47.752+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=150) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-16T09:55:47.754+0000] {standard_task_runner.py:63} INFO - Started process 152 to run task
[2025-10-16T09:55:47.753+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'dag2_bash_1', 'scheduled__2025-10-16T09:54:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp7ynm9d7h']
[2025-10-16T09:55:47.756+0000] {standard_task_runner.py:91} INFO - Job 32: Subtask dag2_bash_1
[2025-10-16T09:55:47.911+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-16T09:55:47.942+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T09:54:00+00:00 [running]> on host bb861ccd12fb
[2025-10-16T09:55:48.037+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='dag2_bash_1' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T09:54:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T09:54:00+00:00'
[2025-10-16T09:55:48.039+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T09:55:48.041+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-16T09:55:48.043+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hello bash"']
[2025-10-16T09:55:48.053+0000] {subprocess.py:86} INFO - Output:
[2025-10-16T09:55:48.204+0000] {subprocess.py:93} INFO - hello bash
[2025-10-16T09:55:48.248+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-16T09:55:48.249+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T09:55:48.366+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=dag2_bash_1, run_id=scheduled__2025-10-16T09:54:00+00:00, execution_date=20251016T095400, start_date=20251016T095547, end_date=20251016T095548
[2025-10-16T09:55:48.417+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T09:55:48.434+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-16T09:55:48.550+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
